mkdocs-ldjson-plugin/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup, find_packages
+
+# Read requirements from requirements.txt
+def read_requirements():
+    with open("requirements.txt", encoding="utf-8") as f:
+        return f.read().splitlines()
+
+setup(
+    name="inject_ld_json",
```

Comment:
@ocaisa Maybe it makes sense to throw this in a dedicated repo, and publish it on PyPI, so others can also use this plugin, but good enough to park it in our `EESSI/docs` repo for now